spark/v3.5/spark-runtime/LICENSE

```diff
@@ -203,109 +203,78 @@
 
 --------------------------------------------------------------------------------
 
-This binary artifact contains Apache Avro.
```

Comment:
Please roll back formatting changes so that this can be reviewed more easily. There are a ton of changes here that make maintenance much harder. For example, if a dependency is moved and reformatted here, it takes time to validate that it is or is not still present in order to see if any action needs to be taken for NOTICE content.

Reply:
Agree

Reply:
I thought it's what you asked using `|`.